src/aclk/mqtt_websockets/ws_client.c

```diff
@@ -263,7 +245,7 @@ int ws_client_start_handshake(ws_client *client)
     if (rbuf_bytes_available(client->buf_read) < x)                                                                    \
         return WS_CLIENT_NEED_MORE_BYTES;
 
-#define MAX_HTTP_LINE_LENGTH 1024*4
+#define MAX_HTTP_LINE_LENGTH (1024 * 4)
```

Comment:
A suggestion: we could use directly 4096 instead to do a multiplication considering this is a well known value in different OS.

Reply:
I will do in a future PR